test/testdata/rbi/fileutils.rb

```diff
@@ -0,0 +1,7 @@
+FileUtils.cd('/')  # change directory
```

Comment:
Your tests are going to fail unless you add an explicit `# typed:` sigil to this file.

Also, this test doesn't actually test your new RBI change unless you make it at least `# typed: true` (the normal Sorbet default would be `# typed: false` if a file doesn't mention a sigil).

Finally, note that tests for RBI changes are optional as long as the diff is simple enough.